recipes/flatcc/all/conanfile.py

```diff
@@ -99,12 +98,24 @@ def package(self):
                       os.path.join(self.package_folder, "bin", "flatcc"))
         # Copy license file
         self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
+        # Copy file with cmake functions for end user
+        self.copy("FlatccGenerateSources.cmake", dst="cmake")
 
     def package_info(self):
-        bin_path = os.path.join(self.package_folder, "bin")
-        self.output.info('Appending PATH environment variable: %s' % bin_path)
-        self.env_info.PATH.append(bin_path)
         debug_suffix = "_d" if self.settings.build_type == "Debug" else ""
+        #flatcc package provides two components: the flatcc compiler binary and the runtime library
         if not self.options.runtime_lib_only:
-            self.cpp_info.libs.append("flatcc%s" % debug_suffix)
-        self.cpp_info.libs.append("flatccrt%s" % debug_suffix)
+            self.cpp_info.components["flatcc_exe"].names["cmake_find_package"] = "flatcc_exe"
+            self.cpp_info.components["flatcc_exe"].libs = ["flatcc%s" % debug_suffix]
```

Comment:
If `flatcc_exe` is the compiler binary, shouldn't `.libs` be empty then?

Reply:
Yes, I will add an extra component for the library that the compiler binary links to.